pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+[id="module-metadata-in-pantheon_{context}"]
+= Module metadata in {ProductShortName}
+
+Each module in {ProductShortName} has associated metadata for search engine optimization (SEO). SEO improves the findability of modules.  
+
+There are two types of module-metadata in {ProductShortName}:
+
+* Intrinsic:
```

Comment:
A descrition list makes more sense to me here:

```adoc
Intrinsic:: Metadata that is automatically inferred from the module.
```

Reply:
Agree, this looks to be the better option. Changed.